README.md

```diff
@@ -36,6 +36,7 @@ Stay up to update with new technology and waste all your morning reading everyth
 - [Fatih Arslan](https://github.com/fatih/color)
 - [Martin Angers](https://github.com/PuerkitoBio/goquery)
 - [skratchdot](https://github.com/skratchdot/open-golang)
+- [Corey Prak](https://www.instagram.com/prakattak/)
```

Comment:
This section is mainly for the dependencies I use on the project. Your name will show up in the release. 👍

Reply:
Ah, definitely. I'll add a credit to stretchr maybe?